ext/reflection/php_reflection.stub.php

```diff
@@ -112,6 +112,10 @@ public function hasTentativeReturnType(): bool {}
     public function getTentativeReturnType(): ?ReflectionType {}
 
     public function getAttributes(?string $name = null, int $flags = 0): array {}
+
+    public function hasParameter(int|string $parameter): bool;
```

Comment:
Here and below:

```suggestion
    public function hasParameter(int|string $param): bool;
```

This is the consistent param name with the other similar params.

Reply:
Although I see that this suggestion is not consistent with the method name... 😅

Reply:
> Although I see that this suggestion is not consistent with the method name... 😅

It's definitely the name used in the `ReflectionParameter` constructor, though `parameter` is used in the method names. Will update it.